src/test.c

```diff
@@ -15,11 +15,23 @@ int main( const char *argv[], int argc )
 {
 	int i, j;
 
+    int wd = w_create(399, 398, 160, 16);
```

Comment:
Give us a new user-level program for testing graphics that creates several sub-windows and draws some simple animation in each one..